src/accountLinks.js

```diff
@@ -0,0 +1,60 @@
+const { createLink, validateLink } = require('3id-blockchain-utils')
+
+function convertToCaip10(address) {
+  let [accountAddress, chainId] = address.split('@')
+  if (!chainId) chainId = 'eip155:1'
```

Comment:
We can actually get the chainId (the number) from the provider: https://github.com/ethereum/wiki/wiki/JSON-RPC#net_version